lib/rack/tempfile_reaper.rb

```diff
@@ -14,7 +14,7 @@ def call(env)
       env['rack.tempfiles'] ||= []
       status, headers, body = @app.call(env)
       body_proxy = BodyProxy.new(body) do
-        env['rack.tempfiles'].each { |f| f.close! } unless env['rack.tempfiles'].nil?
+        env['rack.tempfiles'].each(&:close!)
```

Comment:
Since the env hash is mutable, doesn't it mean that `@app.call` could delete the key?